rrule-wasm/src/lib.rs

```diff
@@ -0,0 +1,19 @@
+use rrule::RRuleSet;
```

Comment:
If we decide to have this in a new crate, then we need to add clippy lints here.

Reply:
just added